src/main/java/nl/ordina/jobcrawler/repo/VacancyRepository.java

```diff
@@ -6,18 +6,14 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.query.Param;
-import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 import org.springframework.stereotype.Repository;
-import org.springframework.web.bind.annotation.CrossOrigin;
 
 import javax.transaction.Transactional;
 import java.util.Optional;
 import java.util.Set;
 import java.util.UUID;
 
 @Repository
-@RepositoryRestResource
```

Comment:
is it not a repositoryRestResource anymore? Why?

Reply:
Because we decided we will be writing our own implementation

Reply:
instead of letting this annotation do it for us